packages/fiori-freestyle/src/index.ts

```diff
@@ -85,4 +91,30 @@ async function generate<T>(basePath: string, data: FreestyleApp<T>, fs?: Editor)
     return fs;
 }
 
+/**
+ * Copies the templates at the specified folder, taking care of version specific folders.
+ * @param tmplPath
+ * @param basePath
+ * @param ffApp
+ * @param fs
+ */
+function copyTemplates(tmplPath: string, basePath: string, ffApp: FreestyleApp<unknown>, fs: Editor) {
+    // Remove odata versions specific path while copying template files
+    const replaceVer = `\\/v${Object.values(OdataVersion).join('|\\/v')}`;
+    const removeVersionTmplPath = (path: string): string => path.replace(new RegExp(replaceVer), '');
+    // Ignore other odata version specific template folders
+    const ignoreFolderPattern = Object.values(OdataVersion).filter((ver) => ver !== ffApp.service?.version);
+    // By template type
+    fs.copyTpl(
+        tmplPath,
+        basePath,
+        ffApp,
+        {},
+        {
+            globOptions: { ignore: ignoreFolderPattern.map((folder) => `**/v${folder}/**`) },
```

Comment:
```suggestion
            globOptions: { ignore: ignoreFolderPattern.map((folder) => `**/${folder}/**`) },
```
Typo?

Reply:
Where?

Reply:
As dicussed this `v` is expected as the v for the version in the folder 👍